static/app/views/starfish/components/sampleEvents.tsx

```diff
@@ -0,0 +1,100 @@
+import Duration from 'sentry/components/duration';
+import GridEditable, {GridColumnHeader} from 'sentry/components/gridEditable';
+import Link from 'sentry/components/links/link';
+import EventView from 'sentry/utils/discover/eventView';
+import {
+  DiscoverQueryProps,
+  useGenericDiscoverQuery,
+} from 'sentry/utils/discover/genericDiscoverQuery';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import {TextAlignLeft} from 'sentry/views/starfish/modules/APIModule/endpointTable';
+
+type Props = {
+  eventView: EventView;
+};
+
+type DataRow = {
+  id: string;
+  'transaction.duration': number;
+};
+
+const COLUMN_ORDER = [
```

Comment:
Small nit, but you can type this nicely
```suggestion
type Keys = 'id' | 'transaction.duration'
type TableColumnHeader = GridColumnHeader<Keys>;


const COLUMN_ORDER: TableColumnHeader[] = [
```

Reply:
👌🏻